### ISSUE TITLE
Adding additional configs for fine-tuning

### ISSUE THREAD

**Issue by sdhossain** [criticalml-uw/TamperBench#64]
We mostly support this already, but making an issue if some more edits to the `FinetuneConfig` are needed.

Comment from Cas: 

> Does tamper bench support easy usage of learning rate warm-ups or gradient clipping?

^ should be a few lines of code to add.
